src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -216,7 +216,10 @@ class SourcesTree extends Component {
     }
 
     if (depth === 0) {
-      return <Svg name="domain" />;
+      return <Svg name="domain" 
+       className={classnames({
+          debuggee: (this.props.debuggeeUrl != null && item.name != null && this.props.debuggeeUrl.indexOf(item.name) !== -1) ? true : false
```

Comment:
This can be simplified to:

```js
debuggee: this.props.debuggeeUrl && item.name && this.props.debuggeeUrl.includes(item.name),
```
You don't need the ternary that does `true : false` since your condition is already a boolean.

Reply:
You do need the `!= null` checks to avoid Flow complaining about sketchy existence checks.

Reply:
@nt1m Thanks for your comment, I thought it would be smarter to use indexOf since it's supported in all the browsers and includes is only supported in the following browsers (according to w3school):
![browser support](https://user-images.githubusercontent.com/31850821/31862098-6e140bb2-b706-11e7-957c-02d42159f6d8.JPG)

Reply:
We use `includes` in other areas of the code base, you can feel safe using it :). We are preferring readable code to compatibility with old browsers. The reason for this is we are updating along with Firefox, which is currently at version 56, going on to 57, and includes is available there. We also support recent versions of chrome, but supporting anything from  years ago does not make sense given the scope of the project. It is great that you are thinking of this and is a really useful skill in doing front end!

Reply:
@codehag Thanks for your clarification!